packages/pwa-theme-venia/package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "@magento/pwa-theme-venia",
+  "name": "@magento/pwa-theme-base",
```

Comment:
Should we hold off renaming the package until the next release? Technically, this is still the Venia theme.